baton/handler.go

```diff
@@ -27,5 +30,7 @@ func serveWs(w http.ResponseWriter, r *http.Request) {
 	c := conn{send: make(chan []byte, 256), ws: ws}
 	h.register <- c
 	go c.writePump()
+	c.send <- []byte(strconv.Itoa(i))
```

Comment:
If we end up doing more complicated initialization (we almost certainly will once the concept of users appears), this will become a json object, but for now, 1 expected value doesn't merit json.